processing/src/main/java/org/apache/druid/frame/util/DurableStorageUtils.java

```diff
@@ -150,4 +154,23 @@ public static String getControllerTaskIdWithPrefixFromPath(String path)
       return null;
     }
   }
+
+  /**
```

Comment:
Can you update the doc with an example of the query results path as well?

Reply:
Updated with examples